src/QualityScore.h

```diff
@@ -8,6 +8,7 @@
 
 #include <string>
 #include <vector>
+#include "translator/definitions.h"
```

Comment:
This file can be removed [as I mentioned in my comment](https://github.com/browsermt/bergamot-translator/pull/109/files?file-filters%5B%5D=.h#r623949653).

Reply:
I'll leave removing this file to you in the subsequent pull request? Once you merge it you can remove the unnecessary files.